### ISSUE TITLE
Didn't find op for builtin opcode REVERSE_V2 version '1'.

### ISSUE THREAD

**Issue by panhu** [tensorflow/tensorflow#59042]
**System information**
- Linux
- TensorFlow installed from source:
- TensorFlow version 

**Provide the text output from tflite_convert**

```
Testing TestInvoke
Didn't find op for builtin opcode 'REVERSE_V2' version '1'. An older version of this builtin might be supported. Are you
 using an old TFLite binary with a newer model?

Failed to get registration from op code REVERSE_V2
```

**Standalone code to reproduce the issue** 
when i want to Invoke use tflite-micro,got a error:

Testing TestInvoke
Didn't find op for builtin opcode 'REVERSE_V2' version '1'. An older version of this builtin might be supported. Are you
 using an old TFLite binary with a newer model?

Failed to get registration from op code REVERSE_V2

can you tell me the reason？ thanks

**Comment by tilakrayal** [comment]
@panhu,
Could you please elaborate on the request. Also, to expedite the trouble-shooting process, could you please provide a minimal code snippet and the TensorFlow version you are using. Thank you!

**Comment by panhu** [comment]
ok, I want to use tflite-micro to transplant my model. my model like this:

        time_dat = Input(batch_shape=(self.batch_size, None))
        # calculate STFT
        real,imag = Stla()(time_dat)
        #real,imag = Lambda(self.stftLayer,arguments = {"x":time_dat,'mode':'real_imag'})
        # normalizing log magnitude stfts to get more robust against level variations
        #real = Lambda(reshape,arguments={'axis':[self.batch_size,-1,self.block_len // 2 + 1,1]})(real)
        #imag = Lambda(reshape,arguments={'axis':[self.batch_size,-1,self.block_len // 2 + 1,1]})(imag)
        real = tf.reshape(real,[self.batch_size,-1,self.block_len // 2 + 1,1])
        imag = tf.reshape(imag,[self.batch_size,-1,self.block_len // 2 + 1,1])

        input_complex_spec = Concatenate(axis = -1)([real,imag])
        '''encoder'''

        if self.input_norm == 'iLN':    
            input_complex_spec = LayerNormalization(axis = [-1,-2], name = 'input_norm')(input_complex_spec)
        elif self.input_norm == 'BN':    
            input_complex_spec =BatchNormalization(name = 'input_norm')(input_complex_spec)
        
        # causal padding [1,0],[0,2]
        conv_1 = Conv2D(32, (2,5),(1,2),name = name+'_conv_1',padding = [[0,0],[1,0],[0,2],[0,0]])(input_complex_spec)
        bn_1 = BatchNormalization(name = name+'_bn_1')(conv_1)
        out_1 = PReLU(shared_axes=[1,2])(bn_1)
        # causal padding [1,0],[0,1]
        conv_2 = Conv2D(32, (2,3),(1,2),name = name+'_conv_2',padding = [[0,0],[1,0],[0,1],[0,0]])(out_1)
        bn_2 = BatchNormalization(name = name+'_bn_2')(conv_2)
        out_2 = PReLU(shared_axes=[1,2])(bn_2)
        # causal padding [1,0],[1,1]
        conv_3 = Conv2D(32, (2,3),(1,1),name = name+'_conv_3',padding = [[0,0],[1,0],[1,1],[0,0]])(out_2)
        bn_3 = BatchNormalization(name = name+'_bn_3')(conv_3)
        out_3 = PReLU(shared_axes=[1,2])(bn_3)
        # causal padding [1,0],[1,1]
        conv_4 = Conv2D(64, (2,3),(1,1),name = name+'_conv_4',padding = [[0,0],[1,0],[1,1],[0,0]])(out_3)
        bn_4 = BatchNormalization(name = name+'_bn_4')(conv_4)
        out_4 = PReLU(shared_axes=[1,2])(bn_4)
        # causal padding [1,0],[1,1]
        conv_5 = Conv2D(128, (2,3),(1,1),name = name+'_conv_5',padding = [[0,0],[1,0],[1,1],[0,0]])(out_4)
        bn_5 = BatchNormalization(name = name+'_bn_5')(conv_5)
        out_5 = PReLU(shared_axes=[1,2])(bn_5)
        
        dp_in = out_5
        
        for i in range(self.numDP):
            
            dp_in = DprnnBlock(numUnits = self.numUnits, batch_size = self.batch_size, L = -1,width = 50,channel = 128, causal=True)(dp_in)#self.DPRNN_kernal(dp_in,str(i),last_dp = 0)
       
        dp_out = dp_in
        
        '''decoder'''
        skipcon_1 = Concatenate(axis = -1)([out_5,dp_out])

        deconv_1 = Conv2DTranspose(64,(2,3),(1,1),name = name+'_dconv_1',padding = 'same')(skipcon_1)
        dbn_1 = BatchNormalization(name = name+'_dbn_1')(deconv_1)
        dout_1 = PReLU(shared_axes=[1,2])(dbn_1)

        skipcon_2 = Concatenate(axis = -1)([out_4,dout_1])
        
        deconv_2 = Conv2DTranspose(32,(2,3),(1,1),name = name+'_dconv_2',padding = 'same')(skipcon_2)
        dbn_2 = BatchNormalization(name = name+'_dbn_2')(deconv_2)
        dout_2 = PReLU(shared_axes=[1,2])(dbn_2)
        
        skipcon_3 = Concatenate(axis = -1)([out_3,dout_2])
        
        deconv_3 = Conv2DTranspose(32,(2,3),(1,1),name = name+'_dconv_3',padding = 'same')(skipcon_3)
        dbn_3 = BatchNormalization(name = name+'_dbn_3')(deconv_3)
        dout_3 = PReLU(shared_axes=[1,2])(dbn_3)
        
        skipcon_4 = Concatenate(axis = -1)([out_2,dout_3])

        deconv_4 = Conv2DTranspose(32,(2,3),(1,2),name = name+'_dconv_4',padding = 'same')(skipcon_4)
        dbn_4 = BatchNormalization(name = name+'_dbn_4')(deconv_4)
        dout_4 = PReLU(shared_axes=[1,2])(dbn_4)
        
        skipcon_5 = Concatenate(axis = -1)([out_1,dout_4])
        
        deconv_5 = Conv2DTranspose(2,(2,5),(1,2),name = name+'_dconv_5',padding = 'valid')(skipcon_5)
        
        '''no activation'''        
        deconv_5 = deconv_5[:,:-1,:-2]

        #output_mask = Activation('tanh')(dbn_5)
        output_mask = deconv_5

        #enh_spec = Lambda(self.mk_mask)([real,imag,output_mask])
        enh_spec = self.mk_mask([real,imag,output_mask])

        self.enh_real, self.enh_imag = enh_spec[0],enh_spec[1]
        
        enh_frame = Lambda(self.ifftLayer,arguments = {'mode':'real_imag'})(enh_spec)
        enh_frame = enh_frame * self.win
        #enh_time = Lambda(self.overlapAddLayer, name = 'enhanced_time')(enh_frame)
        enh_time = self.overlapAddLayer(enh_frame)

        self.model = Model(time_dat,enh_time)
        self.model.summary()

        return self.model

When i use tflite-micro to test my model, the error is:
 [Didn't find op for builtin opcode REVERSE_V2 version '1']
(https://github.com/tensorflow/tensorflow/issues/59042#top)

Is it because tflite micro does not support reverse_v2, because I found reverse_v2 in tflite("tensorflow/lite/kernels/reverse.cc"), but I did not find reverse_v2 in tflite-micro. how can i add "reverse.cc" in tflite-micro.

**Comment by mohantym** [comment]
Hi @panhu!

Thanks for sharing your observation . 
Could you provide above code as gist to replicate this issue. I think you might be using older Tensorflow version (<2.8).

Thank you!
